uvls/src/smt.rs

```diff
@@ -186,7 +188,16 @@ async fn find_fixed(
     cancel: CancellationToken,
 ) -> Result<HashMap<ModuleSymbol, SMTValueState>> {
     let mut state = HashMap::new();
-    for (s, v) in initial_model {
+    let mut optionals: HashMap<ModuleSymbol, SMTValueState> = HashMap::new();
+
+    let x = initial_model;
+    for (s, v) in x {
+        let x = base_module.file(s.instance);
+        if let Some(GroupMode::Optional) =
+            x.group_mode(x.parent(s.sym, false).unwrap_or(Symbol::Root))
+        {
+            optionals.insert(s, SMTValueState::FalseOptional);
```

Comment:
why are optional features automatically considered as FalseOptional (same as in line 272)?